toxcore/tox.api.h

```diff
@@ -286,7 +286,10 @@ const FILE_ID_LENGTH              = 32;
  */
 const MAX_FILENAME_LENGTH         = 255;
 
-
+/**
+ * Max size (in bytes) of names queried by toxcore
+ */
+const QUERY_MAX_NAME_SIZE       = 255;
```

Comment:
I don't know what you mean by two empty lines before a block comment, this is formatted like the rest above it. Do you want me to change all lines above as well?

alignment is fixed